intro/http-client/examples/https_client.rs

```diff
@@ -37,6 +37,7 @@ fn get(url: impl AsRef<str>) -> anyhow::Result<()> {
     // 1. create a new EspHttpClient with SSL certificates enabled
     let mut client = EspHttpClient::new(&EspHttpClientConfiguration {
         use_global_ca_store: true,
+        // TODO: Fix this
```

Comment:
What needs fixing here?

Reply:
For some reason, `esp_idf_sys::esp_crt_bundle_attach` (is on the next line) was not found. I could not investigate it too much tbh, will come back at it next week.

Reply:
Gotcha thanks!

Reply:
I've created https://github.com/esp-rs/espressif-trainings/issues/136 so I don't forget about it